moncli/entities/column_value/complex.py

```diff
@@ -64,6 +74,34 @@ class TimezoneValue(ColumnValue):
     pass
 
 
-class WeekValue(ColumnValue):
+class WeekValue(ComplexNullValue):
     """A week column value."""
-    pass
\ No newline at end of file
+    
+    native_type = Week
+    allow_casts = (dict)
+    
+    
+    def _convert(self, value):
+        try:
+            start_date = datetime.strptime(value['startDate'], DATE_FORMAT)
+            end_date = datetime.strptime(value['endDate'], DATE_FORMAT)
+            return Week(start=start_date, end=end_date)
+        except (KeyError,ValueError):
+            return {}
+        
+    def _cast(self, value):
+        try:
+            start = value['start']
+            end = value['end']
+            if not (start.tzinfo and end.tzinfo):
```

Comment:
This check seems strange to me.  What happens if the condition was false? Do we return nothing instead?  That does not seem right.  I would suggest for simplicity that you do an _astimezone_ call when setting the _start_ and _end_ variables using datetime.now().tzinfo to ensure a local timezone conversion.